### PR TITLE
bt-276: [ BUG FIX ] sign up page layout needs improvement

### LINKED ISSUE #276: [BUG] The layout of the page does not correspond to the approved design of "Sign Up" page
**Description:**

The layout of the page does not correspond to the approved design of "Sign Up" page

**Environment:**
OS - Windows 
Browser - Google Chrome 
Version - 116.0.5845.141

**Steps to reproduce:**

1. Open the page http://localhost:3000/sign-up
3. Enter invalid password in the field “Password” (123@4)
4. Pay attention on  the field “Password”
5. Pay attention on the page layout

**Actual Result:**

1. The validation error message overlap radio button  with the label “I’m hiring” and a radio button with the label “I’m looking for a job”
2. The page contain text in the lower left corner
3. The page contain left sidebar 
4. The icon of the logo is located in the content area without the logo image. The logo image is located in the upper left corner
5. The page contain header 

**Expected Result:**

1. The validation error message  appears under the field “Password”, does not overlap radio button  with the label “I’m hiring” and a radio button with the label “I’m looking for a job”
2. The page must not contain text in the lower left corner
3. The page must not contain left sidebar
4. The logo is located in the content area
5. The page must not have header

**Visual Proof (screenshots, videos, text):**

**Severity:**

Major (S3)

IMG_081

### LINKED ISSUE #292: Message You've been logged in appears without logging in [BUG]
**Description:**
Message You've been logged in appears even if you typed a wrong password and you have a validation message error.

**Environment:**
Win 11, Chrome 116

**Steps to reproduce:**
1. Fill in correct email and not the correct pass
2. Submit
3. Observe the popup and user status

**Actual Result:**
Message appears

**Expected Result:**
Message about log in doesn't appear when you're not logged in actually

**Visual Proof (screenshots, videos, text):**
Image

**Severity:**
Minor (S4)

### PR DESCRIPTION
**Pull request by JosuerBague**
Closes #276

This contains changes that affect the Input component (Cause of the error message bug described in ticket).

Removed dummy text.
Add Logo.
Separated auth routing from being app child route. 

Desktop:
image

Mobile:
image

### CODE REVIEW

**Review by vladjsp — APPROVED** [review]
Looks good. Tested it locally with and without new Terms checkbox.

**Review by Monyster — CHANGES_REQUESTED** [review]
Resizing inputs when an error occurs
image

If you made changes I ask for below, inputs will be corrected

File: frontend/src/bundles/auth/components/auth-forms/sign-up-form/sign-up-form.tsx [hunk 1]
```diff
@@ -53,7 +53,10 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
                 <p className="header">Sign Up to get started!</p>
 
                 <FormControl
-                    className={getValidClassNames('inputContainer', 'email')}
+                    className={getValidClassNames(
+                        'input-container',
```
Comment (Monyster) [inline frontend/src/bundles/auth/components/auth-forms/sign-up-form/sign-up-form.tsx]:
```suggestion
                        'inputContainer',
```

File: frontend/src/bundles/auth/components/auth-forms/sign-up-form/sign-up-form.tsx [hunk 2]
```diff
@@ -64,7 +67,10 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
                     />
                 </FormControl>
                 <FormControl
-                    className={getValidClassNames('inputContainer', 'password')}
+                    className={getValidClassNames(
+                        'input-container',
```
Comment (Monyster) [inline frontend/src/bundles/auth/components/auth-forms/sign-up-form/sign-up-form.tsx]:
```suggestion
                        'inputContainer',
```

**Review by Monyster — CHANGES_REQUESTED** [review]

**Comment by JosuerBague** [comment]
> Resizing inputs when an error occurs image
> 
> If you made changes I ask for below, inputs will be corrected

Yep, made these changes before classNames ticket got merged. Should be okay now.

**Review by Monyster — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**